docs/release-notes/release-notes-0-17-12.md

```diff
@@ -39,8 +39,11 @@ Python SDK
 ### Akamai EdgeWorker Support Updates - Preview
 
 - Delete functionality improved (Dino to add detail here)
-- Can now invoke EdgeWorkers with stream workers. In Samples, sample-function-worker added.
-- Macrometa SDK support
+- You can now invoke EdgeWorkers with stream workers. [Edge Functions](../cep/reference/extensions/execution/edge-functions.md) was added as a result of this change.
+  - In Samples, Sample-Function-Worker added.
+- Added API error response codes. [Functions API](link) were updated as a result of this change.
+- `groupIdWithPrefix` has been added to EdgeWorker metadata. This means you should re-initialize any EdgeWorkers with metadata.
```

Comment:
@diana-macrometa it means that you should reinitialise metadata before creating new edgeworkers.